src/AppBundle/Resources/views/themes/app/user/login.html.twig

```diff
@@ -24,7 +24,7 @@
                                 <label for="username">{{ 'ngsite.user.login.username.label'|trans }}:</label>
                                 <input type="text" id="username" class="form-control" name="_username" value="{{ last_username }}" required="required" autofocus="autofocus" autocomplete="on" placeholder="{{ 'ngsite.user.login.username.placeholder'|trans }}" />
                             </div>
-                            <div class="form-group{% if error %} has-error{% endif %}">
```

Comment:
Same as for page-header, I would leave this so styling can be different between a form item with or without an error.